Classes/Image Upload/ImageUploadTableViewController.swift

```diff
@@ -189,14 +189,12 @@ class ImageUploadTableViewController: UITableViewController {
 
                 switch result {
                 case .error:
-                    print("error")
                     ToastManager.showGenericError()
                     self?.setRightBarItemIdle()
 
                 case .success(let link):
-                    print("success")
```

Comment:
Also removed these prints, I felt like if there was a good reason to keep them they should at least be more descriptive. (cc @Sherlouk)

Reply:
Debug messages, forgot to remove!

Reply:
That's what I thought, cool!